python/example_code/ses/ses_replicateidentities/ses_replicateidentities.py

```diff
@@ -0,0 +1,343 @@
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+# snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+# snippet-sourcedescription:[ses_replicateidentities.py demonstrates how to migrate email address and domain identities from one region to another using the Amazon SES API.]
+# snippet-keyword:[Python]
+# snippet-sourcesyntax:[python]
+# snippet-sourcesyntax:[python]
+# snippet-keyword:[AWS SDK for Python (Boto3)]
+# snippet-keyword:[Code Sample]
+# snippet-keyword:[Amazon Simple Email Service]
+# snippet-service:[ses]
+# snippet-sourcetype:[full-example]
+# snippet-sourcedate:[2020-07-07]
+# snippet-sourceauthor:[nimbid]
+# snippet-start:[ses.python.ses_replicateidentities.complete]
+
+import json
+import sys
+import boto3
+from boto3 import Session
+from botocore.exceptions import ClientError
+S = Session()
+# Returns list of regions where SES is available.
+regions = S.get_available_regions('ses')
+print ("This is the list of available regions:")
+print (regions)
+
+
+# Calls SES API to trigger a verification email.
+def email_verify(email_list, region):
+    ses_dest_client = S.client('ses', region_name=region)
+    for email in email_list:
+        response = ses_dest_client.\
+                   verify_email_identity(EmailAddress=email)
+
+
+# Calls SES API to generate a verification token for the domain being verified.
+def domain_verify(domain_list, region):
+    ses_dest_client = S.client('ses', region_name=region)
+    token_list = []
+    for domain in domain_list:
+        response = ses_dest_client.verify_domain_identity(Domain=domain)
+        token_list.append(response['VerificationToken'])
+    verification_table = dict(zip(domain_list, token_list))
+    # print(verificationTable)
+    return verification_table
+
+# Calls Route53 API to add a TXT record with the verification token for all
+# domains in the verification table.
+def add_route_53_record(table='', rec_type='', dkim_dom='', dkim_tok=''):
+    r53 = S.client('route53')
```

Comment:
Instead of creating the Route 53 client in the function, create it in your main() function and pass it as a param so this function can be unit tested.

Reply:
Done.